packages/core/test/logger.js

```diff
@@ -270,7 +331,7 @@ describe('logger', () => {
     const logger = createlogger({ bundle }, options);
 
     const data = {
-      response_json: {
+      whatever: {
```

Comment:
wanted to make sure these didn't make assumptions about the fact that the response was present in parsed form